Benchmark failure

```
$ go test -bench .
goos: linux
goarch: amd64
pkg: capnproto.org/go/capnp/v3/rpc
cpu: 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
BenchmarkPingPong-8   	--- FAIL: BenchmarkPingPong-8
    level0_test.go:2077: conn error: rpc: send release: send message: transport: stream transport: send: context canceled
    level0_test.go:2077: conn error: rpc: remote abort: connection closed
    level0_test.go:2077: conn error: rpc: send release: send message: transport: stream transport: send: context canceled
    level0_test.go:2077: conn error: rpc: remote abort: connection closed
    level0_test.go:2077: conn error: rpc: send release: send message: transport: stream transport: send: context canceled
    level0_test.go:2077: conn error: rpc: remote abort: connection closed
    level0_test.go:2077: conn error: rpc: incoming call: answer ID 0 reused
    level0_test.go:2077: conn error: rpc: remote abort: rpc: incoming call: answer ID 0 reused
    bench_test.go:48: call failed on iteration 43774: test.capnp:PingPong.echoNum: rpc: connection closed
FAIL
exit status 1
FAIL	capnproto.org/go/capnp/v3/rpc	0.962s
```

Haven't started debugging yet. It occurs to me that CI doesn't run the benchmarks, but it probably should just to make sure they *pass* -- it won't give us super useful data since I don't think github actions is perf isolated enough, but it is not really surprising to me that the benchmarks are broken.

--- a/rpc/bench_test.go
+++ b/rpc/bench_test.go
@@ -2,18 +2,18 @@ package rpc_test
 
 import (
 	"context"
+	"net"
 	"testing"
 
 	"capnproto.org/go/capnp/v3"
 	"capnproto.org/go/capnp/v3/rpc"
 	testcp "capnproto.org/go/capnp/v3/rpc/internal/testcapnp"
-	"capnproto.org/go/capnp/v3/rpc/transport"
 )
 
 func BenchmarkPingPong(b *testing.B) {
-	p1, p2 := transport.NewPipe(1)
+	p1, p2 := net.Pipe()
 	srv := testcp.PingPong_ServerToClient(pingPongServer{})
-	conn1 := rpc.NewConn(rpc.NewTransport(p2), &rpc.Options{
+	conn1 := rpc.NewConn(rpc.NewStreamTransport(p2), &rpc.Options{
 		ErrorReporter:   testErrorReporter{tb: b},
 		BootstrapClient: capnp.Client(srv),
 	})
@@ -23,7 +23,7 @@ func BenchmarkPingPong(b *testing.B) {
 			b.Error("conn1.Close:", err)
 		}
 	}()
-	conn2 := rpc.NewConn(rpc.NewTransport(p1), &rpc.Options{
+	conn2 := rpc.NewConn(rpc.NewStreamTransport(p1), &rpc.Options{
 		ErrorReporter: testErrorReporter{tb: b},
 	})
 	defer func() {

